pkgs/applications/video/handbrake/default.nix

```diff
@@ -99,6 +93,7 @@ stdenv.mkDerivation rec {
     # Force using nixpkgs dependencies
     sed -i '/MODULES += contrib/d' make/include/main.defs
     sed -i '/PKG_CONFIG_PATH=/d' gtk/module.rules
+    sed -i 's/^[[:space:]]*\(meson\|ninja\)[[:space:]]*= ToolProbe.*$//g' make/configure.py
```

Comment:
Could this be made for cmake as well?

Reply:
Also possibly for nasm.

Reply:
Removing nasm followed the same pattern. cmake involved a larger edit that I'm not sure will be as well received. Let me know.